clients/drcachesim/tracer/instr_counter.cpp

```diff
@@ -72,28 +78,73 @@ static uint64 instr_count;
 // but to avoid the complexity of different instrumentation we need a threshold.
 #define DELAY_FOREVER_THRESHOLD (1024 * 1024 * 1024)
 
-static std::atomic<bool> reached_trace_after_instrs;
+static std::atomic<bool> reached_trace_after_instrs(false);
```

Comment:
Was it a bug before that this was not initialized? Aren't atomic bools init to false automatically?

Reply:
Not a bug, only because it's a global, otherwise it would be undefined.
It's zeroed-initialized regardless, but it reads better IMO.
I thought it was a bug during my testing, but the issue I had turned out to be unrelated.

Reply:
LGTM